emails/tests/utils_tests.py

```diff
@@ -59,3 +65,13 @@ def test_generate_relay_From_with_linebreak_chars(self):
             expected_encoded_display_name, '<%s>' % self.relay_from
         )
         assert formatted_from_address == expected_formatted_from
+
+    @override_settings(ON_HEROKU=True, SITE_ORIGIN='https://test.com')
+    def test_get_email_domain_from_settings_on_heroku(self):
+        email_network_locality = get_email_domain_from_settings()
+        assert 'mail.test.com' == email_network_locality
+
+    @override_settings(ON_HEROKU=False, SITE_ORIGIN='https://test.com')
+    def test_get_email_domain_from_settings_not_on_heroku(self):
+        email_network_locality = get_email_domain_from_settings()
+        assert 'test.com' == email_network_locality
```

Comment:
`locality` is a bit confusing here? Can it be just `email_domain`

Reply:
Done 2ad82db